internal/config/config.go

```diff
@@ -359,3 +359,32 @@ func GetC4GHKey() (*[32]byte, error) {
 	keyFile.Close()
 	return &key, nil
 }
+
+// GetC4GHPublicKey reads the c4gh public key
+func GetC4GHPublicKey() (*[32]byte, error) {
+	keyPath := viper.GetString("c4gh.newpublickey")
+
+	// Make sure the key path and passphrase is valid
+	keyFile, err := os.Open(keyPath)
+	if err != nil {
+		return nil, err
+	}
+
+	key, err := keys.ReadPublicKey(keyFile)
+	if err != nil {
+		return nil, err
+	}
+
+	keyFile.Close()
+	return &key, nil
+}
+
+// CheckHeader reads the config and returns if the header will be copied
+func CheckHeader() bool {
+	header := viper.GetString("backup.header")
+	if header == "true" {
+		return true
+	} else {
+		return false
+	}
+}
```

Comment:
```suggestion
// CheckHeader reads the config and returns if the header will be copied
func CopyHeader() bool {
	if viper.IsSet("backup.copyHeader") {
		return viper.GetBool("backup.copyHeader")
	}
	
	return false
}
```

Viper can do direct string to boolean conversions for most cases true/false, 1/0 and so on.

Reply:
Cool, I did not know that and I did not want to put in config something else but strings. This solved it.